ftdetect/sml.vim

```diff
@@ -1,2 +1,5 @@
 " SML signature files
-au BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup VimBetterSMLFtDetect
+  autocmd!
+  autocmd BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup END
```

Comment:
I was under the impression that this was unnecessary, because the file is in a `ftdetect/` folder. At least, that's the way I interpret this entry from `:help ftdetect`:

> Note that there is no "augroup" command, this has already been when sourcing your file.